README.md

```diff
@@ -1,38 +1,160 @@
-# Header
+# VTEX Header
 
-Header is a canonical component that any VTEX app can import.
+## Description
+
+The VTEX Header app is a store component that represents a top fixed navgation bar, and is used by store theme.
+
+:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
 
 ## Release schedule
-| Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
-| :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
-| [1.x]    | **Maintenance LTS** |  2018-09-18     | 2018-11-08            | March 2019  | 1.x
-| [2.x]    | **Current Release** |  2018-11-08     |                       |             | 2.x
 
-To import it into your code:
+| Release |       Status        | Initial Release | Maintenance LTS Start | End-of-life | Store Compatibility |
+| :-----: | :-----------------: | :-------------: | :-------------------: | :---------: | :-----------------: |
+|  [2.x]  | **Current Release** |   2018-11-08    |                       |             |         2.x         |
+|  [1.x]  | **Maintenance LTS** |   2018-09-18    |      2018-11-08       | March 2019  |         1.x         |
+
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
+
+## Table of Contents
+
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
+- [Troubleshooting](#troubleshooting)
+- [Tests](#tests)
+
+## Usage
+
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
+
```

Comment:
This is missing:

We add the header as a block in our [Store](https://github.com/vtex-apps/store/blob/master/store/interfaces.json).